kuberos_test.go

```diff
@@ -107,11 +107,11 @@ func TestPopulateUser(t *testing.T) {
 					"b": &api.Cluster{Server: "https://example.net", CertificateAuthorityData: []byte("PAM")},
 				},
 				Contexts: map[string]*api.Context{
-					"a": &api.Context{AuthInfo: templateUser, Cluster: "a"},
-					"b": &api.Context{AuthInfo: templateUser, Cluster: "b"},
+					"a": &api.Context{AuthInfo: "", Cluster: "a"},
```

Comment:
I think it would be a little clearer what was happening in this test if you set `Username: "example@example.org"` (or whatever you like!) in `extractor.OIDCAuthenticationParams` above. In that case you could check that the `AuthInfo` (aka user) was named `"example@example.org"` rather than the empty string.